src/storage/redis_db.cc

```diff
@@ -322,10 +327,12 @@ rocksdb::Status Database::Keys(const std::string &prefix, std::vector<std::strin
   return rocksdb::Status::OK();
 }
 
-rocksdb::Status Database::Scan(const std::string &cursor, uint64_t limit, const std::string &prefix,
-                               std::vector<std::string> *keys, std::string *end_cursor) {
+rocksdb::Status Database::Scan(const std::string &cursor, uint64_t scan_success_limit, uint64_t scan_false_limit,
+                               const std::string &fix, std::vector<std::string> *keys, std::string *end_cursor,
+                               const int pm) {
```

Comment:
Can you change the param `fix` to a more meaningful name? Can you keep the name the same as the header file?